command.c

```diff
@@ -1274,6 +1280,9 @@ void redis_parse_cmd(struct cmd *r) {
                     }
                     state = SW_ARGN_LEN;
                 } else if (redis_argeval(r)) {
+                    // hiredis-cluster needs atleast one key in eval
```

Comment:
I like this comment from the test case. Add it here too!

                        // Command layout: eval <script> <no of keys> <keys..> <args..>

Also:

```suggestion
                    // hiredis-cluster needs at least one key in eval
```

Reply:
Added this, and added some more details